### ISSUE TITLE
Incorrect assignment of elements

### ISSUE THREAD

**Issue by ErikMarklund** [BradyAJohnston/MolecularNodes#25]
I have a little hydrogen peroxide pdb file that looks like this:
```
ATOM      1  O   PER     0       0.038  -0.113  -0.046  0.00  0.00           O  
ATOM      2  H   PER     0       0.132  -0.803  -0.752  0.00  0.00           H  
ATOM      3  O   PER     0      -1.074  -0.427   0.586  0.00  0.00           O  
ATOM      4  H   PER     0      -0.777  -0.749   1.475  0.00  0.00           H  
END
```
When I load it with MolecularNodes it seems to become three oxygens and one hydrogen. I notice that the auxllary object ohoh_properties_1 only has three vertices, where I would have expected four. The x-values of the vertices are 8.0, 1.0, and 8.0, which I assume are the atomic numbers. Nothing seems to tell Blender that the last atom is a hydrogen. I spotted a potential too short range in create_properties_model, in which case one should remove '-1' from the following:
`locations=list(map(lambda x: [get_value(prop_x, x), get_value(prop_y, x), get_value(prop_z, x)], range(len(first_model.atoms()) - 1))))
`

**Comment by ErikMarklund** [comment]
Update: Adding another vertex to ohoh_properties_1 and putting it at 1,0,0 makes the last hydrogen look like a hydrogen.

**Comment by BradyAJohnston** [comment]
Thank you for the bug report, and for already coming up with the solution as well!

This has been my first python project so I'm still getting the hang of switching between indexing from 0 and 1 etc, so this is one of those examples. I have only been testing with protein structures so haven't noticed the last atom being incorrect. I will fix it for a new release!

**Comment by BradyAJohnston** [comment]
Thanks for the encouragement! Issue should now be fixed in [v0.5.21](https://github.com/BradyAJohnston/MolecularNodes/releases/tag/v0.5.21)
